frontend/src/components/OrganizationSearch.tsx

```diff
@@ -65,6 +64,7 @@ export const OrganizationSearch: React.FC<OrganizationSearchProps> = ({
   // const [tags, setTags] = useState<OrganizationTag[]>([]);
```

Comment:
Can go ahead and remove unused comments and code. (ie. lines 3 - 4, 63 - 64, 135 - 137)
```suggestion
```

Reply:
Unused comments have been removed.